src/Functions/IFunction.cpp

```diff
@@ -310,7 +311,10 @@ ColumnPtr IExecutableFunction::executeWithoutSparseColumns(const ColumnsWithType
 
 ColumnPtr IExecutableFunction::execute(const ColumnsWithTypeAndName & arguments, const DataTypePtr & result_type, size_t input_rows_count, bool dry_run) const
 {
-    if (useDefaultImplementationForSparseColumns())
+    bool use_default_implementation_for_sparse_columns = useDefaultImplementationForSparseColumns();
+    /// DataTypeFunction does not supports obtaining default (isDefaultAt())
```

Comment:
Why don't return a default function, something like ignore?

```
milovidov-desktop :) SELECT ignore()

SELECT ignore()

Query id: fd69092d-62be-42ab-8c63-752def7e7dd1

┌─ignore()─┐
│        0 │
└──────────┘
```

Reply:
It is not interchangeable - https://github.com/ClickHouse/ClickHouse/blob/b38d4b5b0fb6956242e9a870ea5d6275210f3a22/src/Functions/IFunction.cpp#L361-L368